scargo/file_generators/templates/docker/Dockerfile-esp32.j2

```diff
@@ -1,4 +1,7 @@
-RUN  apt -y install flex bison gperf python3-setuptools ninja-build ccache libffi-dev libssl-dev dfu-util libusb-1.0-0
+# esp-idf dependencies, https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/linux-macos-setup.html
+RUN apt update --fix-missing && apt -y --no-install-recommends install git wget flex bison gperf python3 python3-venv cmake ninja-build ccache libffi-dev libssl-dev dfu-util libusb-1.0-0 \
```

Comment:
we already install python3.8 in base layer, meybe we can align single python version for all targets?

Reply:
Here I've just copied the list from the documentation. This way it can be easily updated in the future, when the requirements listed in the documentation change. There's some overlap, but it's not like they'll be installed twice (and it's not a different version, so there's nothing to "align").